components/steps/Status.tsx

```diff
@@ -167,7 +168,7 @@ const StatusStep: React.FunctionComponent<Props> = props => {
                   ? "Congrats, you have been accepted to HackSC 2021. Please confirm/decline your attendance by January 1st"
                   : "Come back soon and see your results."}
               </p>
-              {profileStage === 3 && (
```

Comment:
i think this should still be 3... because 4 means their status is either confirmed or declined, but here we are giving them the option to confirm or decline